### DISCUSSION TITLE
Back to active development and maintenance

### DISCUSSION THREAD

**Discussion by jaysnm** [jaysnm/dremio-arrow#10] [Ideas]
Dear user,

I hope you found this project useful. While it is expected that the package is compatible with most (if not all) dremio engines, a lot has changed in the dremio world. For this reason, I am back to actively maintain this package to keep it useful and reliable.

Regards,
Jason

**Comment by rxm7706** [comment]
@jaysnm thanks for your response !

I think perhaps the first things that come to mind are 

1) Review and update the contributions guidelines, so that everyone aligns to your coding standards and vision.
2) setup dependabot to manage package updates.
3) Maybe open up discussions, so that community can engage with ideas and comments
4) It also helps if you write / create a roadmap using projects - all the features, capabilities, wish lists you would like to see dremio-arrow to have. It will give others an insight into the possibilities and start picking up features to add.
5) We can also look at code coverage - and see if any work needs to be done there - which makes dremio-arrow ready for auto package updates and community contributions. 

Let me know if you think different, have other ideas.

**Comment by jaysnm** [comment]
Dear @rxm7706 

Thank you for your valuable insights. I am deeply thankful for your insights and clear direction. I look forward to having the five bullets enacted. Feel free to put more info on the bullet you are comfortable with.

I have opened discussions panel for the community to engage, thus closing bullet 3.
